README.md

```diff
@@ -32,6 +32,8 @@ The Build Hugo version aligns with [Hugo's releases](https://github.com/gohugoio
       uses: lowply/build-hugo@v0.68.3
 ```
 
+NOTE: As of recently the mirrors for [debian:buster-slim](https://hub.docker.com/_/debian/) are broken. Therefore this action will not work for the versions that reference that base image. A current workaround is to use 0.148.1 or later which uses the `debian:bookworm-slim` base image.
```

Comment:
How about using the [Alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) syntax for callout:


```suggestion
> [!WARNING]
> As of recently the mirrors for [debian:buster-slim](https://hub.docker.com/_/debian/) are broken. Therefore this action will not work for the versions that reference that base image. A current workaround is to use 0.148.1 or later which uses the `debian:bookworm-slim` base image.
```

This should render like this 👇 

> [!WARNING]
> This is a warning message.